### DISCUSSION TITLE
Performance banks

### DISCUSSION THREAD

**Discussion by Piattica** [probonopd/MiniDexed#569] [General]
Is it possible to add many banks of performance in a card or just one?

**Comment by Banana71** [comment]
Currently only one folder with a maximum of 256 performances is supported.  I would also be happy about switchable folders, or something similar🤩

**Reply by probonopd** [reply]
Looks like we need LSB + MSB for performances just like we have for voices so that we can have more than 256.

**Reply by diyelectromusic** [reply]
We need to sort out the performance numbering first... ;)

(see [my latest blog post](https://diyelectromusic.wordpress.com/2023/11/22/a-dx7-usb-dongle/) for the issues I had configuring an instrument file for MIDIOx for use with MiniDexed performances!)

I'll get round to it at some point... but I'm not expecting an easy fix as the performance system has no concept of bank at all at present.

I suggest we capture any further thoughts on the issue here: https://github.com/probonopd/MiniDexed/issues/533

Kevin

**Reply by Piattica** [reply]
@Banana71 
... or a menu that allows to go in a subdir of "Performance"

**Reply by Banana71** [reply]
new thoughts see here: #533
